website/scripts/source-reference.js

```diff
@@ -165,9 +165,9 @@ function moduleNameToFullName(name) {
       return 'In-App Messaging';
     case 'messaging':
       return 'Cloud Messaging';
-      case 'ml':
-        return 'ML';
-      case 'perf':
+    case 'ml':
+      return 'ML';
+    case 'perf':
```

Comment:
I am certain that was my whitespace error, sorry for leaving it + thank you for fixing it!